src/components/votes/votesTab.js

```diff
@@ -112,11 +147,17 @@ VotesTab.propTypes = {
   votes: PropTypes.arrayOf(PropTypes.shape({
     username: PropTypes.string,
     address: PropTypes.string,
+    rank: PropTypes.number,
+    productivity: PropTypes.number,
+    rewards: PropTypes.string,
+    vote: PropTypes.string,
   })),
   loading: PropTypes.array,
   t: PropTypes.func,
+  searchVotesDelegate: PropTypes.func,
```

Comment:
Do you think could be possible change this function names, I believe I know what this do, but it is more clear, **searchForDelegateVotes** or **searchDelegateVotes**, I think it is more clear in this way IMPO

Reply:
This one is kind of tricky, because it's not for getting the votes of a delegate, it's for fetching data of the voted delegate, to make it clearer would need to maybe be **fetchVoteDelegateInfo**